From 04af63d7fecc250ff0e2cbdcf9b86d54408966fd Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 9 Feb 2023 19:21:15 -0800
Subject: [PATCH] parser.h: Including missing <cstdint> (#602)

- Update copyright year

- This is needed from gcc-13+ see [1]

- Fixes

../git/framework/platform/parser.h:282:43: error: 'uint32_t' does not name a type
  282 |                 *type       = static_cast<uint32_t>(number);
      |                                           ^~~~~~~~
../git/framework/platform/parser.h:21:1: note: 'uint32_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?

[1] https://www.gnu.org/software/gcc/gcc-13/porting_to.html

Upstream-Status: Backport [https://github.com/KhronosGroup/Vulkan-Samples/commit/04af63d7fecc250ff0e2cbdcf9b86d54408966fd]
Signed-off-by: Khem Raj <raj.khem@gmail.com>

s
---
 framework/platform/parser.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/framework/platform/parser.h b/framework/platform/parser.h
index 215f3d80..1d0ab23c 100644
--- a/framework/platform/parser.h
+++ b/framework/platform/parser.h
@@ -1,4 +1,4 @@
-/* Copyright (c) 2021, Arm Limited and Contributors
+/* Copyright (c) 2021-2023, Arm Limited and Contributors
  *
  * SPDX-License-Identifier: Apache-2.0
  *
@@ -18,6 +18,7 @@
 #pragma once
 
 #include <cassert>
+#include <cstdint>
 #include <string>
 #include <typeindex>
 #include <vector>
-- 
2.34.1

